src/components/panel/index.md

```diff
@@ -8,9 +8,34 @@ layout: layout-pane.njk
 ---
 
 {% from "_example.njk" import example %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
```

Comment:
Removing all WCAG 2.2 messages for panel, as this would not be a fail of Redundant entry

```suggestion
```